docs/sources/developers/plugins/variables.md

```diff
@@ -0,0 +1,60 @@
++++
+title = "Interpolate variables in plugins"
+type = "docs"
++++
+
+# Interpolate variables in plugins
+
+Variables are placeholders for values, and can be used to create things like templated queries. For more information on variables, refer to [Templates and variables]({{< relref "../../variables/templates-and-variables.md" >}}).
+
+This guide explains how to leverage template variables in your panel plugins and data source plugins.
+
+We'll see how you can turn a string like this:
+
+```sql
+SELECT * FROM services WHERE id = "$service"
+```
+
+into
+
+```sql
+SELECT * FROM services WHERE id = "auth-api"
+```
+
+Grafana provides a couple of helper functions to interpolate variables in a string template. Let's see how you can use them in your plugin.
+
+## Panels
```

Comment:
This heading and the next both seem very vague. Can you make them more specific? WHAT about panels? Support panel variables? Add support for variables in panels to your plugin? I really don't know what this section is about based on the headings.